src/strategies/schedule.py

```diff
@@ -132,7 +132,8 @@ def collect_future_gc_info(self, dt=timedelta(days=1)):
 
         gc_info = [{
             "current_loads": {},
-            "target": gc.target
+            "target": gc.target,
+            "charge": False
```

Comment:
gc.window is based solely on the priorities of the power grid's operator (Prio 1, 2 are CHARGE windows, Prio 3, 4 are not). These priorities are based on the situation of the power grid only (load, curtailment). The schedule's core goal is however not to serve those priorities but to charge the vehicles to their desired SoC while respecting the priorities of the grid operator as much as possible. Thereforem, in certain situations the schedule may ask the fleet owner to charge during a NON-CHARGE window since the core goal of charging the vehicle cannot be met otherwise.